torch/csrc/utils/python_arg_parser.cpp

```diff
@@ -135,9 +135,17 @@ auto handle_torch_function(PythonArgs &r, PyObject* args, PyObject* kwargs, PyOb
   py::object torch_api_function = PyObject_FastGetAttrString(torch_api, (char*)r.get_func_name().c_str());
   TORCH_INTERNAL_ASSERT(torch_api_function.ptr() != nullptr, "torch API function must exist");
   py::object ret;
+
+  // overloaded_args already all have unique types
+  std::vector<py::object> overloaded_types(r.signature.overloaded_args.size());
```

Comment:
I pre-allocated the `std::vector` here so that a lower amount of allocations happen.